src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -458,11 +458,11 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator {
     new ApiResponse(code = 500, message = "Rawls Internal Error")
   ))
   def copyMethodConfigurationRoute = cookie("iPlanetDirectoryPro") { securityTokenCookie =>
-    path("workspaces" / Segment / Segment / "methodconfigs" / "copy" ) { (workspaceNamespace, workspaceName) =>
+    path("workspaces" / Segment / Segment / "methodconfigs" / Segment / Segment / "copy" ) { (workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigurationName) =>
       post {
         entity(as[MethodConfigurationName]) { srcMethodConfigurationName =>
           requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor),
-            WorkspaceService.CopyMethodConfiguration(workspaceNamespace, workspaceName, srcMethodConfigurationName))
+            WorkspaceService.CopyMethodConfiguration(MethodConfigurationName(methodConfigurationNamespace,methodConfigurationName,WorkspaceName(workspaceNamespace, workspaceName)), srcMethodConfigurationName))
```

Comment:
another reason to the uri be workspace independent is it will flag it as something that requires special authorization logic when we get to that. Just about everything else will have the same "do I have access to the workspace" logic but this required read from source and write to dest.